site/_layouts/docs.html

```diff
@@ -15,9 +15,9 @@
         </div>
 
         {% if page.toc %}
-          <div class="d-none d-xl-block col-xl-2 bd-toc">
+          <nav class="d-none d-xl-block col-xl-2 bd-toc">
```

Comment:
Bit conflicted about this. Not every set of links necessarily needs to be wrapped in a `<nav>`. If we were to make this change, we should probably also want to wrap the links in the `<header>` in a `<nav>` ? And, when there are multiple `<nav>`s, you really should start to name them (e.g. using `aria-label="..."`>.

Reply:
Yes I need to name the two navs with aria-label.
Don't need to wrap links in header because they're already in a landmark. The purpose of this commit was to put secondary navigation link in a landmark.